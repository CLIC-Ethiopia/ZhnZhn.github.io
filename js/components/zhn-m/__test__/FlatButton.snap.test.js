"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _FlatButton = _interopRequireDefault(require("../FlatButton"));

var _jsxRuntime = require("react/jsx-runtime");

const _crDfProps = option => ({
  isPrimary: true,
  caption: "Load",
  title: "Click to load",
  hotKey: "l",
  onClick: jest.fn(),
  ...option
});

describe('FlatButton Snap', () => {
  test('should renders correctly with isPrimary true', () => {
    const tree = _reactTestRenderer.default.create( /*#__PURE__*/(0, _jsxRuntime.jsx)(_FlatButton.default, { ..._crDfProps({
        isPrimary: true
      })
    })).toJSON();

    expect(tree).toMatchSnapshot();
  });
  test('should renders correctly with isPrimary false', () => {
    const tree = _reactTestRenderer.default.create( /*#__PURE__*/(0, _jsxRuntime.jsx)(_FlatButton.default, { ..._crDfProps({
        isPrimary: false
      })
    })).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
//# sourceMappingURL=FlatButton.snap.test.js.map