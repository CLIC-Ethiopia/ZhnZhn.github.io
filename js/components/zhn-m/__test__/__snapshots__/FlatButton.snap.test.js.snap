// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlatButton Snap should renders correctly with isPrimary false 1`] = `
<button
  className="bt-flat"
  onClick={[Function]}
  title="Click to load [l]"
>
  <span
    className="bt-flat__caption"
  >
    <span>
      
    </span>
    <span
      style={
        {
          "textDecoration": "underline",
        }
      }
    >
      L
    </span>
    <span>
      oad
    </span>
  </span>
</button>
`;

exports[`FlatButton Snap should renders correctly with isPrimary true 1`] = `
<button
  className="bt-flat"
  onClick={[Function]}
  style={
    {
      "color": "#607d8b",
    }
  }
  title="Click to load [l]"
>
  <span
    className="bt-flat__caption"
  >
    <span>
      
    </span>
    <span
      style={
        {
          "textDecoration": "underline",
        }
      }
    >
      L
    </span>
    <span>
      oad
    </span>
  </span>
</button>
`;
